calicoctl/resourcemgr/resourcemgr.go

```diff
@@ -169,7 +169,8 @@ func (rh resourceHelper) Apply(ctx context.Context, client client.Interface, res
 	// Try to create the resource first.
 	ro, err := rh.Create(ctx, client, resource)
 
-	if _, ok := err.(cerrors.ErrorResourceAlreadyExists); ok {
+	switch err.(type) {
+	case cerrors.ErrorResourceAlreadyExists, cerrors.ErrorOperationNotSupported:
```

Comment:
can you add a comment explaining why we're doing this? Not clear from reading the code why we're doing this

Reply:
i'll add to the comment that was already there shall I?